guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -16,24 +16,38 @@
     "<ul class=\"toc-item\">\n",
```

Comment:
<p>Should we un-bold the words like French and sequences, Entity Recognition purpose. etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
made appropriate changes by un-bolding the words like French, sequences etc.